apps/finance.turtlelabs.turtleswap/manifest.json

```diff
@@ -3,6 +3,13 @@
 	"href": "https://TurtleSwap.TurtleLabs.Finance",
 	"desc": "VeChains Newest Decentralized Exchange, Part of the Turtle Labs DeFi Ecosystem.",
 	"category": "defi",
-	"tags": ["exchange", "dex", "tool", "finance", "utilities"],
+	"tags": [
+    "exchange",
```

Comment:
Indent seems to be incorrect, could you please fix them?

Reply:
Should be fixed now.